packages/flab/stringify.js

```diff
@@ -0,0 +1,9 @@
+console.log(
+	'module.exports = '+
+	require("fs").readFileSync("/dev/stdin", "utf-8")
```

Comment:
Great catch, @roblg.  @gigabo use the `memory-stream` package for that!  Examples:

- mock socket: https://github.com/redfin/react-server/blob/master/packages/react-server/core/__tests__/reactMiddlewareSpec.js
- mock fs: https://github.com/redfin/react-server/blob/master/packages/react-server-cli/src/__tests__/compileClient/compileClientSpec.js